.github/settings.yml

```
@@ -39,7 +39,7 @@ branches:
         require_code_owner_reviews: false
         dismissal_restrictions: {}
       required_status_checks:
-        strict: true
+        strict: false
```

```suggestion
        loose: true
```
I guess it might be something like this to force CI to be green, but not force the branch to be up-to-date with master.
But I am struggling to find examples.